src/shared/dlt_user_shared.h

```diff
@@ -75,6 +75,9 @@
 
 #include <sys/types.h>
 
+// Global variable to check the dlt logging is to a file or to FIFO
```

Comment:
First, if you must use globals, it should still be defined in a .c file so it isn't defined more than once (if the header is used more than once).  

Then, if it is necessary to exchange this information between dlt-user.c and dlt_user_shared.c it might be nicer to provide a function in dlt-user.c that will answer this question (a function that can be called from dlt_user_shared.c)
This is basic encapsulation, and among other things it would "write protect" the variable, I mean prevent other modules than dlt-user from modifying it.